.github/ISSUE_TEMPLATE/40_contributor_documentation.yml

```diff
@@ -0,0 +1,45 @@
+name: Documentation
+description: Report a problem with our documentation
+labels: [ "a:documentation", "to-triage" ]
+assignees: [ ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please search related information in our [latest documentation](https://docs.gradle.org/current/userguide/userguide.html) before opening a documentation issue.
```

Comment:
:question: shouldn't we link to the nightly documentation here as the latest?

Reply:
I'm not sure about this. I think it would be a rare case when it matters, but nightly documentation is a subject of change during the RC phase, right? As a user, I would have an impression that is something unstable and not final. It might be confusing for some folks.

Reply:
Fair enough, resolving